### ISSUE TITLE
Error \hansschouten\phpagebuilder\src\Core\helpers.php:433

### ISSUE THREAD

**Issue by sprklinginfo** [HansSchouten/Laravel-Pagebuilder#51]
after fixing issue [#42 ], I am having another error on line 433:

 Class name must be a valid object or a string

Laravel 7
php 7.3
ubuntu box

**Comment by arnaudbagnis** [comment]
same 
$ php artisan package:discover --ansi
```
 Error 

  Class name must be a valid object or a string

  at vendor/hansschouten/phpagebuilder/src/Core/helpers.php:432
    428▕      */
    429▕     function phpb_active_languages()
    430▕     {
    431▕         $configLanguageCode = phpb_config('general.language');
  ➜ 432▕         $languages = phpb_instance('setting')::get('languages') ?? [$configLanguageCode];
    433▕ 
    434▕         // if the array has numeric indices (which is the default), create a languageCode => languageTranslation structure
    435▕         if (array_values($languages) === $languages) {
    436▕             $newLanguagesStructure = [];

      +16 vendor frames 
  17  [internal]:0
      Illuminate\Foundation\Application::Illuminate\Foundation\{closure}(Object(HansSchouten\LaravelPageBuilder\ServiceProvider))

      +5 vendor frames 
  23  artisan:37
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))

```

**Comment by arnaudbagnis** [comment]
I have fix by replacing line 232 in:
         `$ languages = [$ configLanguageCode];`

**Comment by Gellish** [comment]
![image](https://user-images.githubusercontent.com/40665645/98974437-dddfae00-254f-11eb-9980-001db378e4b9.png)

**Comment by Gellish** [comment]
fresh install laravel 8

**Comment by HansSchouten** [comment]
Hi everyone, this issue has just been fixed in [v0.15.0](https://github.com/HansSchouten/Laravel-Pagebuilder/releases/tag/v0.15.0). Please upgrade your environment.
